client/my-sites/ads/main.jsx

```diff
@@ -88,12 +91,18 @@ const AdsMain = React.createClass( {
 
 	renderInstantActivationToggle: function( component ) {
 		return ( <div>
+			<QueryWordadsStatus siteId={ this.props.site.ID } />
 			<Card className="rads__activate-wrapper">
 				<div className="rads__activate-header">
 					<h2 className="rads__activate-header-title">{ this.translate( 'WordAds Disabled' ) }</h2>
 					<div className="rads__activate-header-toggle">
 						<FormButton
-							disabled={ this.props.site.options.wordads || ( this.props.requestingWordAdsApproval && this.props.wordAdsError === null ) }
+							disabled={
+								this.props.site.options.wordads ||
```

Comment:
As I said - this wont even be rendered: https://github.com/Automattic/wp-calypso/blob/master/client/lib/ads/utils.js#L19

But proptype is always good